packages/plugins/documentation/admin/src/pages/PluginPage/usePluginPage.js

```diff
@@ -0,0 +1,56 @@
+import { useQuery, useMutation, useQueryClient } from 'react-query';
+import { useNotification } from '@strapi/helper-plugin';
+import { fetchData, deleteDoc, regenerateDoc, submit } from './utils/api';
+import getTrad from '../../utils/getTrad';
+
+const useHomePage = () => {
+  const queryClient = useQueryClient();
+  const toggleNotification = useNotification();
+  const { isLoading, data } = useQuery('get-documentation', () => fetchData(toggleNotification));
+
+  const handleError = err => {
+    toggleNotification({
+      type: 'warning',
+      message: err.response.payload.message,
+    });
+  };
```

Comment:
Curious why you decided to name this function while you kept "onSuccess" anonymous? 😛

Reply:
I think both functions should be anonymous since what we care about seeing from a debugging perspective is onSuccess or onError.